internal/index/manager/manager.go

```diff
@@ -349,16 +393,22 @@ nextStateFile:
 			return nil, fmt.Errorf("unable to save state: %w", err)
 		}
 	}
+	mgr.pcapOverIPPackets = make(chan pcapOverIPPacket, 100)
+	mgr.pcapOverIPFlush = make(chan struct{}, 1)
 
 	go func() {
 		for f := range mgr.jobs {
 			f()
 		}
 	}()
 	mgr.jobs <- func() {
+		go mgr.pcapOverIPPacketHandler()
```

Comment:
This doesn't have to be started in a job? Looks weird.

Reply:
no it doesn't, just feels nicer to start it when we are sure that the jobs handler is running already, probably doesn't matter but /shrug